src/components/flowchart/flowchart.test.js

```diff
@@ -281,13 +282,17 @@ describe('FlowChart', () => {
   });
 
   it('applies .parameters class to all parameter nodes', () => {
-    const wrapper = setup.mount(<FlowChart />);
+    const wrapper = setup.mount(<FlowChart />, {
+      beforeLayoutActions: [() => toggleTypeDisabled('parameters', false)],
+    });
     expect(wrapper.render().find('.pipeline-node--parameters').length).toBe(4);
   });
 
   it('applies .parameters class to all the edges from parameter nodes', () => {
-    const wrapper = setup.mount(<FlowChart />);
-    expect(wrapper.render().find('.pipeline-edge--parameters').length).toBe(4);
+    const wrapper = setup.mount(<FlowChart />, {
+      beforeLayoutActions: [() => toggleTypeDisabled('parameters', false)],
```

Comment:
It would be good to add a comment to explain that parameters are enabled here to overwrite default behaviour for better clarity :)